sdks/java/testing/tpcds/src/main/java/org/apache/beam/sdk/tpcds/SqlTransformRunner.java

```diff
@@ -57,6 +64,10 @@
  *
  * <p>TODO: Add tests.
  */
+@SuppressWarnings({
```

Comment:
Please avoid using `SuppressWarnings` on new code. In case there's good reasons / false positives I suggest to annotate the relevant place(s) only